### PR TITLE
Added table layout field

### PR DESCRIPTION
**Pull request by vwinstea**
## Status:
:construction: In development

## Description

The code should be done, I just haven't been able to test it since I've been having issues connecting to the db. Please don't merge until you've tested the api route I wrote!

## Todos

- [ ] Tests

### CODE REVIEW

**Comment by vwinstea** [comment]
@eric-newcomer Here's the PR I mentioned in the meeting today

**Review by eric-newcomer — COMMENTED** [review]
Looks good to me. Not able to test yet, waiting on the login for the member DB console.

File: api/src/api/chapters.js [hunk 1]
```diff
@@ -86,4 +86,39 @@ router.put(
...
+  errorWrap(async (req, res) => {
+    const { memberTableLayout } = req.body;
+    if (!memberTableLayout) {
+      res.status(400).json({
+        success: false,
+        message: 'Table Layout not found',
+      });
+      return;
+    }
+    let chapter = await Chapter.findByIdAndUpdate(
```
Comment (shopaniuk) [inline api/src/api/chapters.js:102]: nit: this can be a const but don't feel like you have to change it

**Comment by shopaniuk** [comment]
Looks good to me!

**Pull request merged**